### PR TITLE
 DScanner added

### PR DESCRIPTION
**Pull request by andre2007**

### CODE REVIEW

**Comment by andre2007** [comment]
Build error seems not related:
```
[2019-07-03T05:18:41.870Z] /home/jenkins/workspace/lugins_warnings-ng-plugin_PR-116@tmp/durable-2cd3f3b0/script.sh: 1: /home/jenkins/workspace/lugins_warnings-ng-plugin_PR-116@tmp/durable-2cd3f3b0/script.sh: Syntax error: "(" unexpected

[2019-07-03T05:18:42.069Z] Terminated

script returned exit code 2
```

File: SUPPORTED-FORMATS.md [hunk 1]
```diff
@@ -120,3 +120,4 @@ the [Analysis Parsers Library](https://github.com/jenkinsci/analysis-model/).
 | 100 | yamllint | yamlLint() | - - | [YamlLint](https://yamllint.readthedocs.io/) |  |
 | 101 | yui | yuiCompressor() | - - | YUI Compressor |  |
 | 102 | zptlint | zptLint() | - - | ZPT Lint |  |
+| 103 | dscanner | dscanner() | - - | DScanner | **/dscanner-report.json |
```
Comment (uhafner) [inline SUPPORTED-FORMATS.md]: Do not edit this file, run the tests in `ToolsLister` to auto-genenerate.

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased](https://github.com/jenkinsci/warnings-ng-plugin/compare/warnings-ng-5.2.0...master)
+- [JENKINS-57098](https://issues.jenkins-ci.org/browse/JENKINS-55051): Added DScanner parser
```
Comment (uhafner) [inline CHANGELOG.md]: Also add a reference to this PR.

File: SUPPORTED-FORMATS.md (hunk 1 above)
Comment (andre2007) [reply SUPPORTED-FORMATS.md]: Done

File: CHANGELOG.md (hunk 2 above)
Comment (andre2007) [reply CHANGELOG.md]: Done

**Comment by uhafner** [comment]
Thanks!

**Pull request merged**